tests/baselines/reference/api/tsserverlibrary.d.ts

```diff
@@ -5754,8 +5754,9 @@ declare namespace ts {
          * @param position A zero-based index of the character where you want the entries
          * @param options An object describing how the request was triggered and what kinds
          * of code actions can be returned with the completions.
+         * @param formattingSettings settings needed for calling formatting functions.
          */
-        getCompletionsAtPosition(fileName: string, position: number, options: GetCompletionsAtPositionOptions | undefined): WithMetadata<CompletionInfo> | undefined;
```

Comment:
I'm guessing this is here accidentally because you were working with my draft PR? same for `typescript.d.ts`

Reply:
Yes, good catch.